chain-cli/network/fablo-config-dev-mode.json

```diff
@@ -21,7 +21,8 @@
           "type": "solo",
           "instances": 1
         }
-      ]
+      ],
+      "tools": { "fabloRest": true }
```

Comment:
yeah, I've tried that before. The case is in dev/watch mode Fabric discovery service does not see deployed chaincodes. And Fablo REST relies on service discovery for connecting to Fabric network. So Fablo REST won't work in dev mode - it won't be able to call chaincodes. If we want to make it work, we need to change Fablo REST source code.